app/uk.gov.hmrc.helptosavestridefrontend/views/partials/form.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.helper.CSRF
+
+@this()
+
+@(
+  action: play.api.mvc.Call,
+  args: (Symbol, String)*)(body: => Html)(implicit request: Request[_]
+)
+
+<form action="@action.url" method="@action.method" @toHtmlArgs(args.toMap[Symbol,String]) novalidate>
```

Comment:
we should be using the uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF component for our forms, we probably don't need a wrapper for that as it can be used in templates usually with a single argument for the action

Reply:
done